src/commands.ts

```diff
@@ -57,5 +59,22 @@ export namespace Commands {
 
     export const JAVA_RESOLVEPATH = "java.resolvePath";
 
+    export const JAVA_PROJECT_GETMAINMETHOD = "java.project.getMainMethod";
+
+    export const JAVA_PROJECT_EXPORTJAR = "java.project.exportJar";
+
     export const VSCODE_OPEN_FOLDER = "vscode.openFolder";
+
+    export const JAVA_BUILD_WORKSPACE = "java.workspace.compile";
+
+    export const JAVA_RESOLVE_BUILD_FILES = "vscode.java.resolveBuildFiles";
```

Comment:
Used anywhere?

Reply:
Used in `resolveBuildFiles` of `jdtls.ts` for fix suggestions when building.